testdata/config/source_ip_deny.yaml

```diff
@@ -0,0 +1,16 @@
+# This rule denies all TCP traffic sent to the  
+# destination service of "foo.default.svc.cluster.local"
+# that is NOT from a source ip addr of "10.11.12.13"
+apiVersion: "config.istio.io/v1alpha2"
+kind: rule
+metadata:
+  name: mixerdenyip
+  namespace: istio-config-default
+  labels:
+    istio-protocol: tcp
+spec:
+  match: source.ip != ip("10.11.12.13") && destination.service == "foo.default.svc.cluster.local"
```

Comment:
we should always put destination.service filter first in an expression.

Reply:
Is this meant to be for perf optimization, or a coding guideline?